src/main/java/dev/openfeature/sdk/MutableContext.java

```diff
@@ -33,7 +33,7 @@ public MutableContext(String targetingKey) {
     }
 
     public MutableContext(Map<String, Value> attributes) {
-        this("", attributes);
+        this("", new HashMap<>(attributes));
```

Comment:
New HashMap is already created in the overloaded constructor.

```suggestion
        this("", attributes);
```